Projects/Car_UltraSonicRefactor/Autonomous_Car/APP/Monitoring/Monitoring.h

```diff
@@ -0,0 +1,23 @@
+/**
+ * @file Monitoring.h
+ * @author your name (you@domain.com)
+ * @brief 
+ * @version 0.1
+ * @date 2021-08-17
+ * 
+ * @copyright Copyright (c) 2021
+ * 
+ */
+
+
+#ifndef MONITORING_H_
+#define MONITORING_H_
+
+
+#include "DataTypes.h"
+#include "Ultrasonic.h"
+
+EN_ERRORSTATE_t Sensor_mainFunction(void);
```

Comment:
I think Mainfunctions should have return type of void (e.g. void Sensor_mainFunction(void);)